Nav.php

```diff
@@ -172,17 +176,23 @@ public function renderItem($item)
         }
 
         if ($items !== null) {
-            $linkOptions['data-toggle'] = 'dropdown';
-            Html::addCssClass($options, 'dropdown');
-            Html::addCssClass($linkOptions, 'dropdown-toggle');
-            if ($this->dropDownCaret !== '') {
-                $label .= ' ' . $this->dropDownCaret;
+            if ($this->asDropdown) {
```

Comment:
This does not seem right.
What if same `Nav` is meant to combine both dropdown and regular list?
I suppose it might be possible, thus 'asDropdown' is better to be a key inside the `$item` array.